gravann/_kappa.py

```diff
@@ -0,0 +1,108 @@
+import torch
+
+from gravann.polyhedral import ACC_L as POLYHEDRAL_ACC_L, U_L as POLYHEDRAL_U_L
+from ._integration import ACC_trap, U_trap_opt
+from ._io import load_sample, load_polyhedral_mesh
+from ._mascon_labels import ACC_L as MASCON_ACC_L, U_L as MASCON_U_L, ACC_L_differential as MASCON_ACC_L_differential
+from ._sample_observation_points import get_target_point_sampler
+
+
+def compute_c_for_model_v2(model, encoding, method, use_acc=True, **kwargs):
```

Comment:
v2?

Reply:
in general feel free to replace / refactor. That is why we are on a fork, so we don't have to worry about breaking compatibility with the original code